packages/contracts/scripts/deploy/manager/DiamondDeployHelper.ts

```diff
@@ -24,8 +24,9 @@ export async function getSelectorsFromFacet(contractName: string, artifactFolder
   return selectors;
 }
 
-export function getContractInstance(contractName: string, account?: any, artifactFolderPath = "../../../out") {
+export function getContractInstance(contractName: string, account?: ethers.Signer | undefined, artifactFolderPath = "../../../out") {
```

Comment:
```suggestion
export function getContractInstance(contractName: string, account?: ethers.Signer, artifactFolderPath = "../../../out") {
```

Code smell. Consider checking the parameter before or after its passed in.